tests/integration/it_locations_test.py

```diff
@@ -1,18 +1,38 @@
+"""Integration tests for Locations"""
 from http import HTTPStatus
+from flask import url_for
+import pytest
 
+from timeless.restaurants.models import Location
 
-def test_list(client):
-    assert client.get("/locations/").status_code == HTTPStatus.OK
 
+def test_list(client, db_session):
+    """ List all locations """
+    db_session.add(__create_location__())
+    db_session.commit()
+    response = client.get(url_for("location.list"))
+    assert response.status_code == HTTPStatus.OK
+    assert b"Test location" in response.data
```

Comment:
@bkuzmic can we also assert that only 1 Location has been listed?

Reply:
@amihaiemil `assert b` means convert string to bytes and compare bytes. I just saw it in the Flask tutorial. I guess it is no mandatory, but if I want to test some special characters it would be needed.